commerce-add-ons/packages/product-feeds/README.md

```diff
@@ -0,0 +1,38 @@
+---
+description: >-
+  Need to generate umbraco commerce product feed? There's a plugin for that.
+---
+
+## Introduction
+After building up your e-commerce website, it's quite common that you want to share your fancy products to a external service like Google Merchant Center to reach out to more viewers.
+
+We have a XML feed generator plugin which simplifies the creation of structured XML files like that. It automatically extracts product data from Umbraco node, organizes it according to standards and allows you to add more information to the feed.
+
+Checkout `Umbraco Commerce Product Feeds plugin`.
+
+Supported feed templates
+- Google Merchant Center.
+
+## Quick start
```

Comment:
1. Please move this content into a new article called "Installation" to match with the other 2 plugins: https://docs.umbraco.com/umbraco-commerce-packages/checkout/installation 
2. Then you can add some content from the overview page of the other packages: https://docs.umbraco.com/umbraco-commerce-packages/deploy/deploy

Reply:
Suggestions were applied.